Sources/Network/ProtocolSpecific/GRPCOnly/TransportProtocol+GRPCTimeout.swift

```diff
@@ -5,7 +5,7 @@ import Foundation
 
 extension TransportProtocol {
     static var grpcTimeout: Double = {
-        Double(GrpcChannelManager.Defaults.callOptionsTimeLimit.timeout!.nanoseconds) / 1.0e9
+        Double(GrpcChannelManager.Defaults.callOptionsTimeLimit.timeout?.nanoseconds) / 1.0e9 ?? 0
```

Comment:
While this is more concise, I like your other approach in "Combined" that reports the error when it's not set:

```suggestion
        guard let timeout = GrpcChannelManager.Defaults.callOptionsTimeLimit.timeout else {
            logger.error("No GrpcTimeout value !")
            return Double(0)
        }
        return Double(timeout.nanoseconds) / 1.0e9
```